src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -142,6 +142,32 @@ OutputVector translate_mul_op(const NodeContext& node) {
     set_node_name(node.get_name(), result);
     return {result};
 }
+
+OutputVector translate_addv2_op(const NodeContext& node) {
+    default_op_checks(node, 2, {}, true);
+    auto lhs = node.get_input(0);
+    auto rhs = node.get_input(1);
+    
+    auto complex_type_mark_lhs = as_type_ptr<ComplexTypeMark>(lhs.get_node_shared_ptr());
+    auto complex_type_mark_rhs = as_type_ptr<ComplexTypeMark>(rhs.get_node_shared_ptr());
+    auto complex_type_inputs = (complex_type_mark_lhs ||  complex_type_mark_rhs) ? true : false;
+    
+    if (complex_type_inputs) {
+        lhs = complex_type_mark_lhs->input_value(0);
+        rhs = complex_type_mark_rhs->input_value(0);
+    }
+
+    auto result = make_shared<v1::Add>(lhs, rhs);
+    if(complex_type_inputs){
+        auto complex_result = make_shared<ComplexTypeMark>(result, complex_type_mark_lhs->get_complex_part_type());
+        set_node_name(node.get_name(), complex_result);
```

Comment:
```suggestion
        set_node_name(node.get_name(), result);
```
It is more meaningful to set node name to `Add` operation instread of marker because marker is eliminated finally.